src/kaby_lake/ddi-translations.cpp

```diff
@@ -0,0 +1,162 @@
+#include "src/kaby_lake/ddi-translations.hpp"
+#include "src/helpers.h"
+
+/* Kabylake H and S */
+static const struct lil_ddi_buf_trans_entry _kbl_trans_dp[] = {
```

Comment:
Can we maybe note the source of these structs? Would be helpful if/when we add more for other gens